### PR TITLE
More metadata

### PR DESCRIPTION
**Pull request by AnikaC-git**
## Reason for changes and summary
Introducing flowsheet metadata and handling for flowsheet row epic id and interface id has changed due to new understanding of these ids. This required changes in the respective tables (was discussed and approved by end users via email/Slack) and has now been implemented.

## Checklist for submitter & reviewer

- [x] copy constructor copies the primary key Id of the entity
- [x] copy constructor copies all other fields of the entity, starting with `super(other)`
- [x] unique constraints added if appropriate
- [x] text definition added to columns which could have more than 255 characters
- [x] foreign key names are in the form `<entityClass>Id`, and no typos
- [x] during run, glowroot is checked for query speed and indexes added if needed
- [x] validation run completed and validated

### CODE REVIEW

**Review by skeating — APPROVED** [review]
Cool - thanks for the comments for the dictionary

**Review by stefpiatek — APPROVED** [review]
🎉

File: inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/visit_recordings/VisitObservationType.java [hunk 1]
```diff
@@ -48,23 +50,27 @@
     private long visitObservationTypeId;
 
     /**
-     * \brief The source system from which we learnt about this visitObservationType.
+     * \brief The ID used in the HL7 messages for referring to this visit observation type.
      *
-     * The hospital system that emap received the data from (e.g. caboodle, clarity, HL7).
+     * This ID in conjunction with the application idInApplication describes the visit observation type uniquely.
```
[the remark this answers is not in the document]
Comment (AnikaC-git) [reply inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/visit_recordings/VisitObservationType.java:55]: added in, thanks

File: inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/visit_recordings/VisitObservation.java [hunk 2]
```diff
@@ -103,6 +104,12 @@
     @Column(columnDefinition = "text")
     private String comment;
 
+    /**
+     * The hospital system that emap received the data from (caboodle or EPIC).
```
Comment (stefpiatek) [inline inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/visit_recordings/VisitObservation.java:108]: we should have the /brief here for user documentation

**Review by stefpiatek — APPROVED** [review]
🎉

**Pull request merged**